### PR TITLE
DSPDC-926 Bump Argo chart and split scripts.

### PR DESCRIPTION
**Pull request by danxmoran**

### CODE REVIEW

File: templates/helm/command-center/templates/argo.yaml [hunk 1]
```diff
@@ -41,10 +41,14 @@ spec:
...
   chart:
-    repository: https://broadinstitute.github.io/monster-helm
-    name: argo-controller
-    version: 0.3.1
+    git: git://github.com/broadinstitute/monster-helm.git
+    ref: dm-argo-27-rc2-DSPDC-926
+    path: charts/argo-controller
+    # repository: https://broadinstitute.github.io/monster-helm
+    # name: argo-controller
+    # version: 0.4.0
```
Comment (danxmoran) [inline templates/helm/command-center/templates/argo.yaml]: Note to self: revert and point at new chart version after broadinstitute/monster-helm#12 merges.

File: templates/helm/command-center/templates/argo.yaml [hunk 2]
```diff
@@ -54,7 +58,17 @@ spec:
...
+      {{ $isDev := eq $env "dev" }}
+      podGarbageCollection:
+        enabled: {{ not $isDev }}
+        {{- if not $isDev }}
+        strategy: OnWorkflowSuccess
+        {{- end }}
+      workflowTTL:
+        enabled: true
+        secondsAfterSuccess: {{ if $isDev }}86400{{ else }}3600{{ end }}
+        secondsAfterFailure: {{ if $isDev }}604800{{ else }}259200{{ end }}
```
Comment (raaidbroad) [inline templates/helm/command-center/templates/argo.yaml]: might be helpful to comment with how many days/hours/minutes these seconds correspond to

File: templates/helm/command-center/templates/argo.yaml [hunk 3]
```diff
@@ -66,10 +80,15 @@ spec:
   targetNamespace: argo-ui
   resetValues: true
   chart:
-    repository: https://broadinstitute.github.io/monster-helm
-    name: argo-server
-    version: 0.4.1
+    git: git://github.com/broadinstitute/monster-helm.git
+    ref: dm-argo-27-rc2-DSPDC-926
```
Comment (raaidbroad) [inline templates/helm/command-center/templates/argo.yaml]: does that note to self you have above apply to this too?

Comment (danxmoran) [reply templates/helm/command-center/templates/argo.yaml]: Ye, updated now

File: templates/helm/command-center/templates/argo.yaml (hunk 2 above)
Comment (danxmoran) [reply templates/helm/command-center/templates/argo.yaml]: Done!

File: templates/helm/orchestration-workflows/templates/clinvar.yaml [hunk 4]
```diff
@@ -18,8 +19,7 @@ spec:
       k8sName: argo-runner
       googleName: clinvar-argo-runner@broad-dsp-monster-{{ .Values.env }}.iam.gserviceaccount.com
     cron:
-      # 8:00am every day for now, as we test.
-      schedule: "0 8 * * *"
+      schedule: {{ .Values.clinvar.schedule }}
```
Comment (quazi-broad) [inline templates/helm/orchestration-workflows/templates/clinvar.yaml]: This is now being passed in as an argument on the hack/apply-orchestration-workflows chart, right?

Comment (quazi-broad) [reply templates/helm/orchestration-workflows/templates/clinvar.yaml]: Ah I see it, line 19. Nvm!

**Pull request merged**